CookingRecipies/Component/signUp.js

```diff
@@ -0,0 +1,89 @@
+import React, {useState} from 'react';
+import {
+  StyleSheet,
+  View,
+  Text,
+  TextInput,
+  TouchableOpacity,
+} from 'react-native';
+import axios from 'axios'
+
+
+
+const SignUp = () => {
+    const [signUp, SetSignUp] = useState({username: '', password: ''})
+
+    console.log(signUp)
+    
+    const onPress = () => {
+     console.log("axios call goes here")
+    //   axios.put('http://www.mocky.io/v2/5da8f1433100005b004e0679', signUp)
+    //   .then(res => console.log('response from sign up axios post', res))
+    //   .catch(err => console.log('error from sign up axios post',err))
+    }
+    
+      return (
+       
+        <View style={styles.signUp}>
+          <Text style={{color: "red"}}>Sign up</Text>
+          <Text>Email</Text>
+          <TextInput
+           style={styles.inputFeilds}
+           value={signUp.username}
+           onChangeText={event => SetSignUp({...signUp, username:event})}/>
+           <Text>Password</Text>
+          <TextInput
+           style={styles.inputFeilds}
+           value={signUp.password}
+           onChangeText={event => SetSignUp({...signUp, password:event})}/>
```

Comment:
the event contains the text the user inputted into the field?  it's not event.target?